lib/aweplug/books/google_books.rb

```diff
@@ -51,12 +51,12 @@ def initialize site, push_to_searchisko
       end
 
       def get data
+        env_param = {
+            :q => "isbn:#{data['isbn']}"}
```

Comment:
Just a style thing, but either put this hash on one line, or the closing `}` on a different line to make it easier to see where it ends.